lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -486,15 +466,13 @@ export class PublicClientApplication {
     /**
      * Use to log out the current user, and redirect the user to the postLogoutRedirectUri.
      * Default behaviour is to redirect the user to `window.location.href`.
+     * @param logoutRequest 
      */
-    logout(account: AccountInfo, authorityString?: string): void {
-        const authorityObj = StringUtils.isEmpty(authorityString) ? this.defaultAuthorityInstance : AuthorityFactory.createInstance(
-            this.config.auth.authority,
-            this.config.system.networkClient
-        );
-
-        // create logout string and navigate user window to logout. Auth module will clear cache.
-        this.authModule.logout(account, authorityObj).then((logoutUri: string) => {
+    logout(logoutRequest: EndSessionRequest): void {
```

Comment:
Do we just want to make `logout` async as well? That way you can use async/await.

Reply:
Also, this seems like something that could have been done incrementally after this PR, in teh interest of a smaller PR.

Reply:
The only async part of login is the authority discovery, so I would rather keep it void.

Reply:
Moving it out of this PR.

Reply:
done.